code-challenges.test.js

```diff
@@ -92,3 +195,35 @@ var vowelTester3 = "challenge"
 
 
 // b) Create the function that makes the test pass.
+
+//create a function that takes a string as an arguement
+//split string into array
+//use indexOf to find the vowel using a variable that stores vowels.
+//return result
+
+//I tried to refactor this many times, went tried to write a for loop (commented below), cannot seem to get this test to pass no matter how I try to refactor it.  I know i am close, but just don't know what i am missing.  Have spent hours on this one and number 3.....
+
+// const vowelFinder = (string) => {
+//   let vowelArray = string.split("")
+//   let vowels = ['a','e','i','o','u']
+//   let firstVowelArray = vowelArray.findIndex(vowels => {
+//     return
+//   })
+//   return firstVowelArray
+// }
+//
+// console.log(vowelFinder(vowelTester1));
+// console.log(vowelFinder(vowelTester2));
+// console.log(vowelFinder(vowelTester3));
+
+const vowelFinder = (string) => {
+  let vowelArray = string.split("")
+  for(let i = 0; i < vowelArray.length; i++){
```

Comment:
consider using `.forEach()` instead of a for-loop. You can still have access to an index by doing something like 
```
vowelArray.forEach((letter, index) => { ... })
```